templates/Header.stencil

```diff
@@ -0,0 +1,9 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
As stated above, I would recommend something in the name that clearly indicates that this isn't intended to be a standalone thing. `HeaderSnippet.stencil` or `HeaderPartial.stencil`.

Reply:
Rename to HeaderSnippet.stencil